i18n/ar/cosmic_term.ftl

```diff
@@ -0,0 +1,100 @@
+cosmic-terminal = طرفية COSMIC
+new-terminal = طرفية جديدة
+
+# Context Pages
+
+## About
+git-description = إيداع جِت {$hash} بتاريخ {$date}
+
+## Color schemes
+color-schemes = مخططات الألوان
+rename = أعِد التسمية
+export = صدِّر
+delete = احذِف
+import = استورِد
+import-errors = تعذَّر الاستيراد
+
+## Profiles
+profiles = اللاحات
+name = الاسم
+command-line = سطر الأوامر
+tab-title = عنوان اللسان
+tab-title-description = يتجاوز عنوان اللسان المبدئي
+add-profile = أضِف لاحة
+new-profile = لاحة جديدة
+make-default = اجعله المبدئي
+working-directory = دليل العمل
+hold = التعليق
+remain-open = إبقائه مفتوحُا بعد مغادرة العمليات البنوية.
+
+## Settings
+settings = الإعدادات
+
+### Appearance
+appearance = المظهر
+theme = السمة
+match-desktop = مطابقة سطح المكتب
+dark = داكن
+light = فاتح
+syntax-dark = صياغة السمة الداكنة
+syntax-light = صياغة السمة الفاتحة
+default-zoom-step = خطوات التكبير
+opacity = عتمة الخلفية
+
+### Font
+font = الخط
+advanced-font-settings = إعدادات الخط المتقدمة
+default-font = الخط
+default-font-size = حجم الخط
+default-font-stretch = تمدد الخط
+default-font-weight = وزن الخط العادي
+default-dim-font-weight = وزن الخط الخافت
+default-bold-font-weight = وزن الخط السميك
+use-bright-bold = جعل النص السميك ساطعًا
+
+### Splits
+splits = الانقسامات
+focus-follow-mouse = تركيز الكتابة يتبع الفأرة
+
+### Advanced
+advanced = متقدم
+show-headerbar = إظهار الترويسة
```

Comment:
```suggestion
show-headerbar = إظهار قائمة النافذة
```
Maybe not the best suggestion, and maybe not 100% accurate. But I hate the neologism used much more.

Same for the line below.